pysmt/optimization/goal.py

```diff
@@ -0,0 +1,51 @@
+from pysmt.shortcuts import Times, Int
```

Comment:
Missing header: all PySMT files need to have the standard header (just as the one in `__init__.py`)